File: code/c/procedure-decl/say-hello-proc.c
/*
* Program: say-hello-proc.c
* Outputs 'Hello' messages to the Terminal.
*/

#include "splashkit.h"

void say_hello()
{
    write_line("Hello...");
}

void say_is_anyone_there()
{
    write_line("Is anyone there?");
}

int main()
{
    say_hello();
    say_hello();
    say_is_anyone_there();

    return 0;
}

File: code/pascal/procedure-decl/SayHelloProc.pas
program SayHelloProc; // Outputs 'Hello' messages to the Terminal.

procedure SayHello();
begin
    WriteLn('Hello...');
end;

procedure SayIsAnyoneThere();
begin
    WriteLn('Is anyone there?');
end;

procedure Main();
begin
    SayHello();
    SayHello();
    SayIsAnyoneThere();
end;

begin
    Main();
end.